Taskwarrior Pomodoro/AppDelegate.swift

```diff
@@ -212,9 +233,61 @@ class AppDelegate: NSObject, NSApplicationDelegate, NSMenuDelegate {
             pendingArguments = [definedDefaultFilter] + pendingArguments
         }
         
+        return getTasksUsingFilter(pendingArguments)
+    }
+    
+    func getTodaysPomodorosLog() -> JSON? {
+        let logFilter = ["status:Completed", kPomodoroLogEntryDescription, "entry:today", "limit:1"]
```

Comment:
When I first read this, I have to admit I said outloud "This seems a little crazy; why would @Yoi-Nami-Ra choose to track that this way", but after reading through how this works, and keeping in mind how this would provide all sorts of tangential benefits for free including logging and synchronization, I've now came to the conclusion that this is actually a very elegant solution.  I'm really very surprised and impressed.